docs/specs/skills-system/skills-syncing.md

```
@@ -0,0 +1,718 @@
+    reason: string
+  }>
+}
+```
+
+---
+
+## 3. 架构设计
+
+### 3.1 模块架构
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)